packages/analytics-js/src/services/StoreManager/component-cookie/index.ts

```diff
@@ -12,33 +12,17 @@ export type CookieOptions = {
 /**
  * Encode.
  */
-const encode = (value: any): string | undefined => {
-  try {
-    return encodeURIComponent(value);
-  } catch (e) {
-    // TODO: should it throw error?
-    console.error('error `encode(%o)` - %o', value, e);
```

Comment:
Why we removed the try catch? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent#exceptions

Reply:
It is not required anymore. As explained in the below comments.